src/components/chip-set/chip-set.scss

```
@@ -167,11 +162,21 @@ limel-icon.mdc-chip__icon.mdc-chip__icon--leading {
 }
 
 .mdc-floating-label {
+    @include looks-like-input-label;
     &.mdc-text-field--required::after {
         content: '*';
     }
-    &.force-float {
+    &.lime-floating-label--float-above,
+    &.lime-floating-label--float-above.mdc-floating-label--float-above {
         @include label--float-above;
+        top: $floating-label-top-value;
+    }
+}
+
+.lime-notched-outline--notched {
+    .mdc-notched-outline__notch {
+        border-top-color: transparent;
+        border-top: 1px solid transparent !important;
```

The `border-top-color` attribute is overwritten by the `border-top` attribute, so it's completely redundant.

```suggestion
        border-top: 1px solid transparent !important;
```

---

good catch!